packages.json

```diff
@@ -203,6 +203,12 @@
     "private": false,
     "path": "packages/libs/wallet-adapter-ecko"
   },
+  {
+    "name": "@kadena/wallet-adapter-chainweaver-legacy",
+    "version": "0.0.1-beta.0",
+    "private": false,
```

Comment:
This one needs to be true for now, I need to update the rest as well. Until we do non-beta releases.